packages/component-base/src/uri-utils.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @license
+ * Copyright (c) 2023 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+
+/**
+ * Check if two paths match
+ *
+ * @param {string} path1
+ * @param {string} path2
+ */
+export function isMatchingPath(path1, path2) {
```

Comment:
Here are some alternative names that we could also consider:

```suggestion
export function matchPaths(path1, path2) {
```

```suggestion
export function pathsMatch(path1, path2) {
```

```suggestion
export function arePathsMatching(path1, path2) {
```

Reply:
I renamed this before based on a [comment](https://github.com/vaadin/web-components/pull/5974#discussion_r1230885270). I guess `arePathsMatching` satifsies both. What do you think?

Reply:
Updated as `arePathsMatching ` for now.